docs/modules/storage/pages/recover-single-member.adoc

```diff
@@ -194,7 +202,7 @@ NOTE: If members have backup reads enabled, some in-memory data may appear to ha
 While the member is down, operations on partitions that are owned by that member will be retried until they either time out or the member restarts and executes the requested operation. As a result, this option is best when you prefer a latency spike rather than migrating data over the network.
 
 If member B does not restart within the rebalance delay, the cluster recovers member B's data from backups and
-redistributes the data among the remaining members (members A and C
+repartitions the data among the remaining members (members A and C
 in this case). If member B is later restarted, it recovers its persisted data from disk and brings it up-to-date with data from members A and C. If Merkle trees are enabled on available data structures, migrations use those to request only missing persisted data. For details about how members use Merkle trees, see <<synchronizing-persisted-data-faster, Synchronizing Persisted Data Faster>>.
 
 == Synchronizing Persisted Data Faster
```

Comment:
Probably unrelated to this PR, but it could be worth adding the info about Merkle trees being implicitly enabled for persistent data structures, unless explicitly disabled https://github.com/hazelcast/hazelcast-enterprise/pull/4283